apigentools/constants.py

```diff
@@ -15,4 +15,5 @@
 GITHUB_REPO_URL_TEMPLATE = "github.com/{{github_org_name}}/{{github_repo_name}}"
 HEADER_FILE_NAME = "header.yaml"
 SHARED_SECTION_NAME = "shared"
-REDACTED_OUT_SECRET = "<apigentools:secret-value-redacted-out>"
\ No newline at end of file
+REDACTED_OUT_SECRET = "<apigentools:secret-value-redacted-out>"
+OPENAPI_JAR = "openapi.jar"
```

Comment:
Maybe it would be better to make this `openapi-generator.jar`, since that's what the upstream jars are named?